src/RouteStackInterface.php

```diff
@@ -6,39 +6,42 @@
 
 use Traversable;
 
+/**
+ * @template TRoute of RouteInterface
+ */
 interface RouteStackInterface extends RouteInterface
 {
     /**
      * Add a route to the stack.
      *
-     * @param  string  $name
-     * @param  mixed   $route
-     * @param  int $priority
-     * @return RouteStackInterface
+     * @param string                   $name
+     * @param array|Traversable|TRoute $route
```

Comment:
`array|Traversable` to `iterable` as above

Reply:
58aa9bfcde673252fe98629c3067838ffe1ba62b